lib/libutils/ext/include/compiler.h

```diff
@@ -20,6 +20,7 @@
 #define __packed	__attribute__((packed))
 #endif
 #define __weak		__attribute__((weak))
+#define __weak_alias(x) __attribute__((weak, alias(x)))
```

Comment:
Thinking about it, I'd prefer `#define __alias(x) __attribute__((alias(x)))`. Reason being, I find the resulting macro ambiguous. Consider `void f(void) __weak_alias("g");`. What is weak? `f()` or `g()`? With `void f(void) __weak __alias("g");` it is clear that we are adding an attribute (the name of the alias) to a weak function.